tutorials/os_czasu_kina.md

```diff
@@ -0,0 +1,91 @@
+# Oś czasu kina
```

Comment:
Do tego nie ma co się doczepić ... ale to rozwiązanie nie przystaje do tego co przeze mnie jest oczekiwaną odpowiedzią na zadanie. Proszę o założenie kolejnego zadania w którym ten podział osi czasu kina zostanie sprecyzowany kodem generującym kolumnę.

Reply:
Zadanie utworzone, link: https://app.clickup.com/t/33fvncd